### PR TITLE
[skip-ci] README.md: rephrase Buildah description

### PR DESCRIPTION
**Pull request by vrothberg**
Rephrase the description of Buildah to make some core attributes
(e.g., rootless, Dockerfile-less build, etc.) more obvious.

Addresses: [#1349 (comment)](https://github.com/containers/buildah/issues/1349#issuecomment-466096550)
Cherry-pick-from: https://github.com/containers/buildah/pull/1362
Signed-off-by: Valentin Rothberg <rothberg@redhat.com>

### CODE REVIEW

**Comment by vrothberg** [comment]
@TomSweeneyRedHat PTAL

**Comment by vrothberg** [comment]
@baude @mheon @rhatdan PTA:

File: README.md [hunk 1]
```diff
@@ -83,18 +83,23 @@ Information about contributing to this project.
...
-with [Buildah](https://buildah.io) [(GitHub)](https://github.com/containers/buildah) and
-[Podman](https://podman.io) [(GitHub)](https://github.com/containers/libpod).  Both Buildah and Podman are
-command line tools that work on OCI images and containers.  The two projects
-differentiate in their specialization.
+Buildah and Podman are two complementary open-source projects that are
+available on most Linux platforms and both projects reside at
+[GitHub.com](https://github.com) with Buildah
+[here](https://github.com/containers/buildah) and Podman
+[here](https://github.com/containers/libpod).  Both, Buildah and Podman are
+command line tools that work on Open Container Initiative (OCI) images images
```
Comment (mheon) [inline README.md]: Images images?

**Comment by mheon** [comment]
Please fix typo

File: README.md (hunk 1 above)
Comment (vrothberg) [reply README.md]: Fixed! Sorry for the silly typo.

Comment (vrothberg) [reply README.md]: Also fixed in Buildah now https://github.com/containers/buildah/pull/1367 (Cc @TomSweeneyRedHat)

Comment (TomSweeneyRedHat) [reply README.md]: and sniff, I reviewed that line in buildah and podman and missed it in both places....  Thanks for the pickup @mheon.

Comment (vrothberg) [reply README.md]: I think the 80 character line breaks make it hard to review as the diffs can be very chaotic. In a previous life, when writing mostly LaTeX, I tried to have one sentence per line which made it easier.

Comment (mheon) [reply README.md]: Agree here - markdown will mangle it into a paragraph no matter how we format it.

**Pull request merged**